docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -2475,26 +2493,33 @@ Subsequently, the bundles can be used at runtime:
 <3> `Lucie` is the parameter of the message bundle interface method.  
 <4> It is also possible to obtain a localized message for a key resolved at runtime using a reserved key `message`. The validation is skipped in this case though.
 
-.Message Bundle Interface Example
+
+==== Default Bundle Name
+
+The bundle name is defaulted unless it's specified with `@MessageBundle#value()`.
+For a top-level class the `msg` value is used by default.
+For a nested class the name starts with `msg` followed by an undercore, followed by the simple names of all enclosing classes in the hierarchy (top-level class goes first) seperated by underscores.
+
+For example, the name of the following message bundle will be defaulted to `msg_Index`:
+
 [source,java]
 ----
-import io.quarkus.qute.i18n.Message;
-import io.quarkus.qute.i18n.MessageBundle;
+class Index {
 
-@MessageBundle <1>
-public interface AppMessages {
+    @MessageBundle
+    interface Bundle {
 
-    @Message("Hello {name}!") <2>
-    String hello_name(String name); <3>
+        @Message("Hello {name}!")
+        String hello(String name);
+   }
 }
 ----
-<1> Denotes a message bundle interface. The bundle name is defaulted to `msg` and is used as a namespace in templates expressions, e.g. `{msg:hello_name}`.
-<2> Each method must be annotated with `@Message`. The value is a qute template. If no value is provided, then a corresponding value from a localized file is taken. If no such file exists an exception is thrown and the build fails.
-<3> The method parameters can be used in the template.
+
+NOTE: The bundle name is also used as a part of the name of a localized file, e.g. `msg_Index` in the `msg_Index_de.properties`.
```

Comment:
Does it fallback to `msg.properties` if the key is not defined in the `msg_Index.properties`?

Reply:
I think that it does fallback to the message from the default bundle (be it a file or a message interface). But I don't remember the details.